### ISSUE TITLE
Reference not detected if lowercase characters

### ISSUE THREAD

**Issue by BiKC** [Illumina/GTCtoVCF#64]
In V 1.2.0, if the reference was "a", and GTC said "A", this wouldn't be a problem. However, in V1.2.1, this is no longer the case and causes problems like 
`10      779284  GSA-rs2486591   g       A,G     .       PASS`

V1.2.0 would have been:
`10      779284  GSA-rs2486591   G       A     .       PASS`

**Comment by jjzieve** [comment]
Ok, thanks for bringing this to our attention. I will look into it.

**Comment by jjzieve** [comment]
Having trouble reproducing this. Your reference genome.fa has lowecase characters, is that correct?
e.g.
```
>1
atcg...
```

Also, which product are you running? GSA version 3?

**Comment by BiKC** [comment]
Indeed version 3, GRCh37. We do use a custom genome file however, that only contain the major chromosomes and mitochondrion. 

The problem we have is that the genome has both upper and lowercase characters, and (what I think) when one of the lowercase letters is compared to the GTC files that have upper case characters, it thinks it isn't the same. This causes the problem in the VCF file which in turn causes problems further downstream our pipeline. 

`agcaaaaagggcctctctgaacagattctcatgctgcctgctatgtcagg
agtaagcaccttctttgtctctgactcaggagtctcaggtcatgctacca
tcatttatgaagttgtgattgctgaacatgttagattgcaaacgagtaaa
caggtcagaccctttacTAAGTTGATACCACTTAATTGCATTCTGAATTC
CTTGTTCTGCAACACTTCAAATGACAGAGGTTTCAGCCTCCAGCTAGATA
TGGACTCTTAAAAAATGTCCTAATCAGAATTCTGTAGACTCTTTTACaca
gaattctgggtacaaacatcctctgtactcagaactttgaatgtacgtgt
atattgtctcctggtactggtgctgaggatgaggattccagaggcttact
attcttttcctgatgtcctttaggtctgtttgttaaagcttttattgttt
tcctcctggatgctttctggtctcctgttttgtacgtggtcttatgcaat`

This is a screenshot of the output we had in version 1.2.0 (left) and 1.2.1 (right) with the exact same steps.
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/56390957/85536892-c856ff80-b613-11ea-92a1-82a13b80e39e.png)

Here is another example of the issue:
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/56390957/85536944-d3119480-b613-11ea-9455-f8b31daa6761.png)

Here an example of the error we get further downstream with an imputation tool called Beagle:
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/56390957/85537220-110eb880-b614-11ea-8a96-f57859f4e17d.png)

**Comment by jjzieve** [comment]
You're correct in that the bug exists if the reference genome is lowercase. I still wasn't able to reproduce it ever working when reverting back to `1.2.0` or even older though. Is it possible a different fasta file was used or something? I just pushed https://github.com/Illumina/GTCtoVCF/tree/bug/fix-lowercase-ref-genome can you confirm if that fixes the issue?
